db/migrate/20170908145201_add_index_to_user_agent.rb

```diff
@@ -0,0 +1,15 @@
+require 'lhm'
+
+class AddIndexToUserAgent < ActiveRecord::Migration
+  def self.up
+    Lhm.change_table :user_agents do |m|
+      m.add_index [:user_agent_string]
+    end
+  end
+
+  def self.down
+    Lhm.change_table :user_agents do |m|
+      m.remove_index [:user_agent_string]
+    end
+  end
+end
```

Comment:
We won't need to use `Lhm` for this migration, as it's adding an index to user_ages which is an empty table at this point.  We can even do it in the  `CreateUserAgents` migration too.